lib/screens/home/accounts/multi_account_item.dart

```diff
@@ -71,6 +61,17 @@ class _MultiAccountItemState extends State<MultiAccountItem> {
 
   @override
   Widget build(BuildContext context) {
+    final bloc = Provider.of<AccountsBloc>(context, listen: false);
+    bloc.updated.listen((e) {
```

Comment:
This could end up listening many times. Consider listening in `didChangeDependencies`, with the default `listen: true`, making sure to first cancel the subscription before listening again.

Same for other instances of listening in build methods.